src/react-native/11-navigation/navigation.md

```diff
@@ -2,23 +2,120 @@
 @parent learn-react-native 11
 @outline 3
 
-@description TODO
+@description Learn how to use navigation to move between screens in a React Native app.
 
 @body
 
 ## Overview
 
 In this section, you will:
 
-- TODO
+- Understand why routing is important
+- Be introduced to deep linking
+- Learn about stack-based routing
+- Understand the navigation lifecycle
+- Implement a bottom tab navigation pattern
+- Learn about the difference between stack and native stack navigation
+- Implement a stack navigation pattern
 
 ## Objective 1: Using React Native bottom tab navigation
 
-TODO
+As our application grows in complexity, more screens will be added. The React Navigation library provides a solution that allows us to move between these screens. In this section, we will cover both common navigation patterns used in both Android and iOS applications: tab navigation and stack navigation.
 
-### Concept TODO
+The first step is to install the React Navigation library. Run the following command in your terminal:
 
-TODO
+```bash
+npm install @react-navigation/native
+```
+
+### What is routing?
+
+Routing in React Native refers to the process of managing navigation between different screens in our application. It allows users to move between different parts of our application by clicking on links, buttons, or other interactive elements.
+
+### Why is routing important?
+
+By introducing several screens in our application, it allows us to break down the user interface into smaller, more manageable parts. This makes our application easier to maintain and scale as it grows.
+
+### Deep Linking
+
+Although out of scope for this training, deep linking is a way to navigate to a specific screen or piece of content within your mobile application from an external source. For example, a user might click on a link in an email that opens a specific screen in your app. Deep linking is a powerful tool for user engagement and retention.
+
+The React Navigation library used in this section supports deep linking and has a [guide](https://reactnavigation.org/docs/deep-linking/) on how to set up your application to be notified of incoming links.
+
+### Stacks
```

Comment:
I would move this down to Objective 2 because we don’t do anything with stacks until then.